megaparse/Converter.py

```diff
@@ -32,45 +34,46 @@ class Converter:
     def __init__(self) -> None:
         pass
 
-    def convert(self, file_path: str) -> str:
+    def convert(self, file_path: Path | str) -> str:
         raise NotImplementedError("Subclasses should implement this method")
 
     def save_md(self, md_content: str, file_path: Path | str) -> None:
         with open(file_path, "w") as f:
             f.write(md_content)
 
+
 class XLSXConverter(Converter):
     def __init__(self) -> None:
         pass
 
-    def convert(self, file_path: str) -> str:
-        xls = pd.ExcelFile(file_path) #type: ignore
+    def convert(self, file_path: Path | str) -> str:
```

Comment:
Small suggestion here : should check that path is valid if passed a str
```suggestion
    def convert(self, file_path: Path | str) -> str:
      if isinstance(file_path, str):
           file_path = Path(file_path) 
```